test/src/api/commands/web-element/testGetAttribute.js

```diff
@@ -38,6 +38,54 @@ describe('element().getAttribute() command', function () {
     assert.strictEqual(resultValue, 'text');
   });
 
+  it('test .element.find().attr()', async function() {
```

Comment:
Can you change both these test cases to be of form `test .element().attr() alias`? The change needs to be done inside the test case as well to reflect the change.

Reply:
Done in 3e55f987b1e494f3a9337bff051cd99aa3d2fbef